packages/@react-stately/slider/src/useSliderState.ts

```diff
@@ -68,8 +74,15 @@ export function useSliderState(props: SliderProps): SliderState {
     props.onChange as any
   );
   const [isDraggings, setDraggings] = useState<boolean[]>(new Array(values.length).fill(false));
+  const [isEditables, setEditables] = useState<boolean[]>(new Array(values.length).fill(true));
```

Comment:
You could consider storing this in a ref instead of state potentially, since updating whether a thumb is editable doesn't need to trigger a re-render.